docs/source/trend_lines.rst

```diff
@@ -0,0 +1,128 @@
+.. _trend_lines:
+
+Trend Lines
+===========
+
+With this application, users can create trend lines from vertices and data.
+
+.. figure:: ./images/trend_lines/trend_intro.png
+            :align: center
+            :width: 100%
+
+
+New user? Visit the `Getting Started <getting_started>`_ page.
+
+
+Application
+-----------
+
+The following sections provide details on the different parameters exposed in the user-interface (``trend_lines.ui.json``).
+
+.. figure:: ./images/trend_lines/trend_lines_ui.png
+            :align: center
+            :width: 300
+
+Data Selection
+^^^^^^^^^^^^^^
+
+ - **Object**: Select the target ``Points`` or ``Curve`` object from the dropdown list.
+ - **Data**: [Optional] Referenced data defining properties to link with segments. If omitted, the segments will be created from vertices alone.
+
+.. _filters:
+
+Connection Filters
+^^^^^^^^^^^^^^^^^^
+ - **Parts**: [Optional] Referenced data defining the unique parts of the object. Connections between vertices belonging to the same part are ignored.
+ - **Minimum edges**: Minimum number of segments for a trend line to be valid.
+ - **Maximum distance**: Maximum distance between two points to consider for trend line detection.
+ - **Damping factor**: Threshold parameter [0, 1] penalizing changes in direction between connected segments.
+    At 0, the direction change is not penalized, and at 1, the direction change is maximally penalized.
+
+
+Orientation Filter
+^^^^^^^^^^^^^^^^^^
+ - **Azimuth**: Filtering of segments orientation based on the azimuth angle.
+ - **Tolerance**: Tolerance on deviation of segments orientation from the reference azimuth angle.
+
+Methodology
+-----------
+
+The conversion from scatter point data to trend lines is achieved by the following steps:
+
+1. The selected ``Points`` or ``Curve`` object's vertices are triangulated to form a Delaunay mesh.
+   The mesh is used to identify the initial edges connecting the vertices.
+
+    .. figure:: ./images/trend_lines/trend_step1.png
+            :align: center
```

Comment:
The images are a nice touch! what is the significance of the blue and red dashed lines?

Reply:
Re-edited for clarity